systemd-udevd/pkg.yaml

```diff
@@ -31,6 +31,7 @@ steps:
         # Already built this
         rm -rf /rootfs/usr/lib/udev/hwdb.d
         rm /rootfs/usr/lib/udev/rules.d/{README,60-cdrom_id.rules,60-persistent-alsa.rules,60-persistent-v4l.rules,64-btrfs.rules,70-joystick.rules,70-mouse.rules,70-touchpad.rules,78-sound-card.rules,90-vconsole.rules,99-systemd.rules}
+        rm -f /rootfs/usr/lib/udev/{cdrom_id,v4l_id}
```

Comment:
@dsseng think these are needed anywhere?

Reply:
Hmm, IIRC those are called by some of the built-in rules on some occasions

Reply:
yeah, I think they are being used